membership_file/tests/tests_model.py

```diff
@@ -15,16 +15,16 @@ class MemberModelTest(TestCase):
     fixtures = ['test_users.json', 'test_members.json']
 
     # Tests whether a member is displayed correctly
-    def test_member_display_method(self):
-        # Member has the correct display method
-        user = User.objects.filter(username="test_member").first()
-        display_str = get_member_display_name(user)
-        self.assertEqual(display_str, Member.objects.filter(user__username="test_member").first().get_full_name())
-
-        # Non-Member has the correct display method
-        user = User.objects.filter(username="test_user").first()
-        display_str = get_member_display_name(user)
-        self.assertEqual(display_str, user.get_simple_display_name())
+    # def test_member_display_method(self):
+    #     # Member has the correct display method
+    #     user = User.objects.filter(username="test_member").first()
+    #     display_str = get_member_display_name(user)
+    #     self.assertEqual(str(user), Member.objects.filter(user__username="test_member").first().get_full_name())
+
+    #     # Non-Member has the correct display method
+    #     user = User.objects.filter(username="test_user").first()
+    #     display_str = get_member_display_name(user)
+    #     self.assertEqual(display_str, user.get_simple_display_name())
 
```

Comment:
Is there a reason why it's commented out?

Reply:
Past-me did not realise that present-me would forget to uncomment and update it 😅

Reply:
Figured, it's very familliar XD